erpnext/hr/doctype/employee_advance/employee_advance.js

```diff
@@ -2,19 +2,19 @@
 // For license information, please see license.txt
 
 frappe.ui.form.on('Employee Advance', {
-	setup: function(frm) {
+	setup: function (frm) {
 		frm.add_fetch("employee", "company", "company");
 		frm.add_fetch("company", "default_employee_advance_account", "advance_account");
 
-		frm.set_query("employee", function() {
+		frm.set_query("employee", function () {
```

Comment:
```suggestion
		frm.set_query("employee", function() {
```

We don't need a space between the call and the function name. Update wherever changed